modules/statics/src/coins.ts

```diff
@@ -630,6 +631,7 @@ export const coins = CoinMap.fromCoins([
   erc20('euroc', 'Euro Coin', 6, '0x1abaea1f7c830bd89acc67ec4af516284b1bc33c', UnderlyingAsset.EUROC),
   erc20('eux', 'EUR Stable Token', 18, '0x1b9064207e8046ec1d8e83de79380ed31283914f', UnderlyingAsset.EUX),
   erc20('ever', 'Everscale', 9, '0x29d578cec46b50fa5c88a99c6a4b70184c062953', UnderlyingAsset.EVER),
+  erc20('evry', 'Evrynet', 18, '0xd7dcd9b99787c619b4d57979521258d1a7267ad7', UnderlyingAsset.EVRY),
```

Comment:
<img width="336" alt="image" src="https://user-images.githubusercontent.com/15767732/214275572-aced487f-8df9-4a2c-91e1-2f947d9a970f.png">


Token name seems to be  **EvrynetToken**

Reply:
Ah, changed now!